doc/source/Resources/pyaedt_installer_from_aedt.py

```diff
@@ -237,7 +234,7 @@ def install_pyaedt():
 
             run_command('"{}" install --no-cache-dir --no-index --find-links={} pyaedt'.format(pip_exe, unzipped_path))
         else:
-            run_command('"{}" --default-timeout=1000 install pyaedt[all]'.format(pip_exe))
```

Comment:
I would keep it, because it is when the zip path does not exist, then it tries to perform an online installation. 

I think this one should be: 
run_command('"{}" --default-timeout=1000 install pyaedt[installer]'.format(pip_exe))